jax/interpreters/partial_eval.py

```diff
@@ -173,22 +173,21 @@ def process_call(self, call_primitive, f: lu.WrappedFun, tracers, params):
 
     if call_primitive in call_partial_eval_rules:
       return call_partial_eval_rules[call_primitive](self, call_primitive, f, tracers, params)
-    in_pvs, in_consts = unzip2([t.pval for t in tracers])
-    fun, aux = partial_eval(f, self, in_pvs)
-    out_flat = call_primitive.bind(fun, *in_consts, **params)
-    out_pvs, jaxpr, env = aux()
-    env_tracers = map(self.full_raise, env)
-    out_pv_consts, consts = split_list(out_flat, [len(out_flat)-len(jaxpr.constvars)])
+
+    jaxpr, out_pvs, consts, env_tracers = self.partial_eval(
```

Comment:
Could you change `out_pvs` to `out_pvals`, and correspondingly change corresponding iteration variables from `pv` to `pval`?

There's a funny convention we've (mostly) stuck to where `pv` is a name for the first component of a PartialVal pair, and `pval` is a name for the full pair.

Reply:
Yeah this has always been super confusing for me, because `pv` sounds like a partial value, but they're really `None` or an abstract value.